docs/multicluster/user-guide.md

```diff
@@ -592,43 +633,149 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
-## Multi-cluster Pod-to-Pod Connectivity
+## Multi-cluster Stretched NetworkPolicy
```

Comment:
How about name it "NetworkPolicy for Cross-cluster Traffic", as "policies to Multi-cluster Services" may not be called "stretched"? Or maybe you can come up with a better name.

Reply:
Done